diagnostic.md

```diff
@@ -6,70 +6,70 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
+// find and update
 ```
 
 In what file does the above code go?
 
 ```md
-// your response here
+// the controller
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+// bundle exec rake will update the gem files and bd:migrate will migrate a database
 ```
 
 What is the difference between `rails g scaffold things` and
 `rails g controller things`?
 
 ```md
-// your response here
+// rails g scaffold will create a whole framework while rails g controller just makes the controller folder
```

Comment:
`rails g scaffold things` will generate a migration, model, routes, serializer, controller, and tests for `things`.
`rails g controller things` will generate a controller and tests for `things`.